### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by alwaaffa** [community/community#31945] [Other Feature Feedback, Questions, & Ideas]
I need help for coding by python about (Community outlier detection in social networks based on graph) #anomaly_detection #outlier_detection #graph #python #community #coding #help
Email: alrajhihajar@gmail.com

**Comment by airtower-luna** [comment]
You're unlikely to get help if you just post a vague topic and then expect people to mail you. It's best if you ask your actual question(s), then people can reply if they have an answer.
